tests/hpc/ww4.pm

```diff
@@ -45,6 +45,13 @@ sub run ($self) {
     record_info "warewulf.conf", script_output("cat /etc/warewulf/warewulf.conf");
     record_info "defaults.conf", script_output("cat /usr/share/warewulf/defaults.conf");
 
+    # Authentication support
+    my $warewulf_oci_username = get_var('HPC_WAREWULF_OCI_USERNAME');
+    if ($warewulf_oci_username) {
+        assert_script_run('export WAREWULF_OCI_USERNAME=' . $warewulf_oci_username);
+        assert_script_run('export WAREWULF_OCI_PASSWORD=' . get_var('HPC_WAREWULF_OCI_PASSWORD', get_var('SCC_REGCODE_HPC')));
```

Comment:
Typing scc code directly to console is not right, it will be saved in openQA logs. What can we do?